DRAFT_slitasje.Rmd

```diff
@@ -0,0 +1,1496 @@
+temp <- naturetypes_wide %>%
+  group_by(var_7TK) %>%
+  summarise(sum = n())
+
+ggplot(temp, aes(x = factor(var_7TK),
+                 y = sum))+
+  geom_bar(stat="identity",
+           fill="grey",
+           colour = "black")+
+  theme_bw(base_size = 12)+
+  labs(x = "7TK score",
+       y = "Number of localities")
+```
+
+Then I can combine these two variables into a composite variable called
+`slitasje`
+
+```{r, warning=F}
+naturetypes_wide$slitasje <- apply(naturetypes_wide[,c("var_7TK", "var_7SE")], 1, max, na.rm=T)
+```
+
+When both variables are NA, we get -Inf. There were nine cases of this
+(in 2023). Removing these now.
+
+```{r}
+naturetypes_wide <- naturetypes_wide[naturetypes_wide$slitasje>=0,]
+```
+
+```{r slitasje-naturetype, fig.cap="Slitasje scores.", fig.width=3}
+temp <- naturetypes_wide %>%
+  group_by(slitasje) %>%
+  summarise(sum = n())
+
+ggplot(temp, aes(x = factor(slitasje),
+                 y = sum))+
+  geom_bar(stat="identity",
+           fill="grey",
+           colour = "black")+
+  theme_bw(base_size = 12)+
+  labs(x = "slitasje score",
+       y = "Number of localities")
+```
+
+I appears most localities are in good condition.
+
+I would also like to know how often 7TK was defining of the
+slitasje-indicator, and how often it was 7SE. I can do this by taking
+the difference
+
+```{r, fig.cap="Counting the number of cases where 7TK or 7SE ws defining of the slitasje indicator score.", fig.width=3}
+diff <- naturetypes_wide$var_7SE - 
+        naturetypes_wide$var_7TK
+
+diff <- ifelse(diff == 0, "7TK and 7SE",
+               ifelse(diff <0, "7TK", "7SE"))
+
+diff_tbl <- as.data.frame(table(diff))
+
+ggplot(diff_tbl, aes(x = diff, y = Freq))+
+  geom_bar(stat = "identity",
+           colour = "black",
+           fill = "grey")+
+  theme_bw(base_size = 12)+
+  labs(x = "Defining variable",
+       y = "Number of localities")
+```
+
+Looks like 7SE is more likely to be in a detrimental state.
+
+Now I will copy these slitasje-values into the sf object.
+
+```{r}
+naturetypes$slitasje <- naturetypes_wide$slitasje[match(naturetypes$identifikasjon_lokalid, naturetypes_wide$identifikasjon_lokalid)]
+#nrow(naturetypes[is.na(naturetypes$slitasje),])  # 13 cases
+naturetypes <- naturetypes[!is.na(naturetypes$slitasje),]
+```
+
+#### GRUK
+
+This variable is also recorded in
+[GRUK](https://www.nina.no/Naturmangfold/Trua-natur/%C3%85pen-grunnlendt-kalkmark).
+The nature type data set I'm working on here includes this data already
+(presently only 2021 included). GRUK also records a related variable:
+`% cover in 5m radii circles`, which si much more detailed. This data is
+not published. In any case it is better to use the harmonized data set
+in our case.
+
+#### ANO {#ANO-import}
+
+Arealrepresentativ Naturovervåking (ANO) consist of 1000 systematically
+placed locations each with 18 sample points. In each sample point a
+circle of 250 m2 is visualised, and the main ecosystem is recorded.
```

Comment:
probably not "visualised" (_synliggjort_) but "mapped" or "inspected"?

Reply:
Visualised I think is correct. It is a hypothetical and imagined circle.